.ado/templates/apple-job-react-native.yml

```diff
@@ -69,6 +69,16 @@ steps:
       xcode_actions: ${{ parameters.xcode_actions_release }}
       xcode_useXcpretty: false
       xcode_destination: ${{ parameters.xcode_destination }}
+  
+  - template: apple-xcode-build.yml
+    parameters:
+      xcode_sdk: ${{ parameters.xcode_sdk }}
+      xcode_configuration: Debug -enableAddressSanitizer YES
+      xcode_workspacePath: packages/rn-tester/RNTesterPods.xcworkspace
+      xcode_scheme: ${{ parameters.xcode_scheme }}
+      xcode_actions: ${{ parameters.xcode_actions_debug }}
```

Comment:
couldn't really tell is it only for debug simulator build for iOS flavor?

Reply:
It's not obvious from code inspection, but the CI logs show that at the bottom of the build/test runs for anyone else curious how to know - https://github.com/microsoft/react-native-macos/runs/5105535330